front/src/pods/master/master.container.tsx

```diff
@@ -109,14 +111,18 @@ export const MasterContainer = () => {
               playerCollectionRef.current,
               payload
             );
-
+            showScreenReaderMessage(
+              `El usuario ${payload} se ha conectado`,
+              1500
```

Comment:
Do you think is necessary to change to 1500 the timeout? Juanjo recommends 1000 milliseconds (1 second)